bcda/auth/plugin/alpha_test.go

```diff
@@ -151,14 +151,20 @@ func (s *AlphaAuthPluginTestSuite) TestRevokeClientCredentials() {
 	}
 	db.Save(&user)
 
-	token, _, _ := s.AuthBackend.CreateToken(user)
+	params := fmt.Sprintf(`{"clientID":"%s", "ttl":720}`, user.AcoID.String())
+	_, err := s.p.GenerateClientCredentials([]byte(params))
```

Comment:
Want to make @dhgreene aware of this, since it will have to get changed with https://github.com/CMSgov/bcda-app/pull/140 if this PR is merged first.